manifests/init.pp

```diff
@@ -27,6 +27,8 @@
   $arch             = $sonarqube::params::arch,
   $https            = {},
   $ldap             = {},
+  # ldap and pam are mutually exclusive. Setting $ldap will annihilate the setting of $pam
```

Comment:
@PierreR I'm not sure to understand this comment. I don't see in the code how ldap can annihilate the setting of pam. Could you please clarify?